### PR TITLE
Updated util-auto-url and added image-path

### PR DESCRIPTION
**Pull request by vincentrouleau**

### CODE REVIEW

File: util-count.xsl [hunk 1]
```diff
@@ -13,7 +13,7 @@
 				<xsl:value-of select="count($items)" />
 			</xsl:when>
 			<xsl:otherwise>
-				0
+				<xsl:text>0</xsl:text>
```
Comment (dominic-blain) [inline util-count.xsl:16]: Why the change for text here?

Comment (vincentrouleau) [reply util-count.xsl:16]: I usually try not to leave "free" text in templates. 
They tend to fuck up the indentation in the dom.

Comment (dominic-blain) [reply util-count.xsl:16]: Yeah I meant compared to keeping it a number.. I worry about our comparisons: If we take this result (string '0') and test "if greater than 0 (number)", what will happen?

Could you use value-of instead?

Comment (dominic-blain) [reply util-count.xsl:16]: @hotdoy Ping! What do you think?

Comment (vincentrouleau) [reply util-count.xsl:16]: I don't see why value-of a string would be better than a text but let's test it and see
